packages/lesswrong/components/ea-forum/auth/EALoginPopover.tsx

```diff
@@ -342,7 +345,7 @@ export const EALoginPopover = ({open, setAction, isSignup, classes}: {
   }, [open]);
 
   const title = isSignup
-    ? "Sign up to get more from the EA Forum"
+    ? `Sign up to get more from the ${forumShortTitleSetting.get() || "forum"}`
```

Comment:
This should probably use the `siteNameWithArticle` setting

Reply:
that'd become `the Effective Altruism Forum` vs `the EA Forum` in that case though. (happy to change it, but was trying to keep existing functionality unchanged)

Reply:
The `siteNameWithArticle` setting does actually (confusingly) use the short name (at least it does on the EA forum).

It's not a big deal - I don't really mind either way, but it might be weird in the future for any forums that don't want to add "the" to their name.

Reply:
Updated it to use `siteNameWithArticle`